test/e2e/scenario/telemetry.scenario.ts

```diff
@@ -16,8 +16,9 @@ describe('telemetry', () => {
         window.DD_LOGS!.logger.log('hop', context as any)
       })
       await flushEvents()
-      expect(serverEvents.telemetry.length).toBe(1)
-      const event = serverEvents.telemetry[0] as TelemetryErrorEvent
+
+      // get the last telemetry event because telemetry configuration may have been collected first
+      const event = serverEvents.telemetry[serverEvents.telemetry.length - 1] as TelemetryErrorEvent
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about having a dedicated utility to return only telemetry errors? something like:
```ts
event = serverEvents.telemetryErrors()[0]
```